packages/malloy/src/malloy.ts

```diff
@@ -891,24 +890,10 @@ export class PreparedQuery implements Taggable {
 
   /**
    * Get the flattened version of a query -- one that does not have a `pipeHead`.
+   * @deprecated Because queries can no longer have `pipeHead`s.
    */
-  public getFlattenedQuery(defaultName: string): PreparedQuery {
-    let structRef = this._query.structRef;
-    if (typeof structRef !== 'string') {
-      structRef = structRef.as || structRef.name;
-    }
-    const turtleDef = flattenQuery(this._modelDef, {
-      ...this._query,
-      type: 'query',
-      name:
-        'as' in this._query ? this._query.as || this._query.name : defaultName,
-    });
-    return new PreparedQuery(
-      {...turtleDef, structRef, type: 'query'},
-      this._modelDef,
-      this.problems,
-      this.name || turtleDef.as || turtleDef.name
-    );
+  public getFlattenedQuery(_defaultName: string): PreparedQuery {
```

Comment:
So it turned out that `flattenQuery` from `malloy_query.ts` also copied filters on the source into the turtle def. I don't think that's desired here, or used at all by the users of this function, but I'm not 100% sure. I'll have to check that the VSCode usage of this function still works.

Reply:
Looks like it's only used by Composer actually